src/sentry/search/events/builder/profile_functions.py

```diff
@@ -45,8 +45,15 @@ def get_field_type(self: ProfileFunctionsQueryBuilderProtocol, field: str) -> st
 
     def process_profiling_function_columns(self, row: SnubaRow):
         if "all_examples()" in row:
+            key = "all_examples()"
+        elif "all_examples" in row:
+            key = "all_examples"
```

Comment:
Where is the non `()` version coming from?

Reply:
This is because in the `events` endpoint, we apply `transform_alias_to_input_format` but no on the `events-stats` endpoint.